stan/math/prim/fun/cholesky_decompose.hpp

```diff
@@ -46,35 +52,36 @@ inline Eigen::Matrix<T, Eigen::Dynamic, Eigen::Dynamic> cholesky_decompose(
  * original matrix \f$A\f$ is given by
  * <p>\f$A = L \times L^T\f$.
  *
- * @param m Symmetrix matrix.
+ * @param m Symmetric matrix.
```

Comment:
Also update docs for T

Reply:
Also instead of T maybe EigMat or something

Reply:
done